main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"os"
+
+	"github.com/codegangsta/cli"
+	"github.com/rancher/convoy-agent/storagepool"
+	"github.com/rancher/convoy-agent/volume"
+)
+
+var (
+	GITCOMMIT = "HEAD"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "convoy-agent"
+	app.Version = GITCOMMIT
+	app.Author = "Rancher Labs"
+	app.Usage = "An agent that acts as an interface between rancher storage and cattle server"
+
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "debug, d",
+			Usage: "enable debug logging level",
+		},
+		cli.StringFlag{
+			Name:   "cattle-url",
```

Comment:
can you change the flags to be `url`, `access-key`, `secret-key`, but still use `CATTLE_URL`, etc for env vars.  This is consistent with other components